assets/js/index.js

```diff
@@ -15,19 +15,20 @@ window.fbAsyncInit = function() {
    fjs.parentNode.insertBefore(js, fjs);
  } (document, 'script', 'facebook-jssdk'));
 
+
```

Comment:
Do we need this extra blank line? Most coding standards frown on more than one blank line.